src/components/ourVision.jsx

```diff
@@ -0,0 +1,118 @@
+import { FaNpm } from "react-icons/fa";
+import { TbBrandVscode } from "react-icons/tb";
+import { RxDotFilled } from "react-icons/rx";
+import { useNavigate } from "react-router-dom";
+
+const OurVision = ({ dataList }) => {
+  const navigate = useNavigate();
+
+  const handleInstallButtonClick = () => {
```

Comment:
Let's replace these with anchor tags to open in a separate tab, these will be external links, not routes within the same site